README.md

```diff
@@ -17,13 +17,26 @@
 Some Android plugin versions have issues with Gradle's build cache feature. When applied to an Android project this plugin applies workarounds for these issues based on the Android plugin and Gradle versions. For other versions, please see [older versions.](#older-android-gradle-plugin-versions)
 
 * Supported Gradle versions: 7.0+
-* Supported Android Gradle Plugin versions: 7.0.4, 7.1.3, 7.2.2, 7.3.1, 7.4.2, 8.0.2, 8.1.0 and 8.2.0-alpha16
+* Supported Android Gradle Plugin versions: 7.0+
 * Supported Kotlin versions: 1.7.0+
 
-We only test against the latest patch versions of each minor version of Android Gradle Plugin.  This means that although it may work perfectly well with an older patch version (say 7.0.1), we do not test against these older patch versions, so the latest patch version is the only version from that minor release that we technically support.
-
 The Android cache fix plugin is compatible with the [Gradle Configuration Cache](https://docs.gradle.org/current/userguide/configuration_cache.html#header) when used in conjunction with Android Gradle Plugin 4.2.0 and above.  Using the configuration cache with earlier versions of the Android Gradle Plugin is not supported.
 
+### Compatibility matrix by AGP version
```

Comment:
I find this matrix and section confusing. Above we already say `Supported Android Gradle Plugin versions: 7.0+` so do we really need this?

Reply:
I found that adding the compatibility table in the first section adds a bit of verbosity to the initial plugin description. 
The initial description contains general build stack tooling information and BT features compatibilities(conf_cache). 

The new subsection contains specific information about our release cycle related to the AGP versions tested and supported by the plugin. 

Don't have a strong preference here and open for suggestions/ideas

Reply:
This does seem quite redundant. The words "compatible" and "supported" are quite similar and it isn't clear where we are using what.

Reply:
compatibility matrix removed